[2017-02-16 11:05:41.041] [ERROR] http - ::1 - - "POST /employeeScore HTTP/1.1" 500 2494 "http://localhost:3000/index"
[2017-02-16 11:06:57.234] [ERROR] http - ::1 - - "POST /employeeScore HTTP/1.1" 500 2494 "http://localhost:3000/index"
[2017-02-16 11:25:42.443] [ERROR] http - ::ffff:127.0.0.1 - - "POST /employeeScore HTTP/1.1" 500 3948 "http://localhost:3000/index"
[2017-02-16 11:26:05.135] [ERROR] http - ::1 - - "POST /employeeScore HTTP/1.1" 500 3948 "http://localhost:3000/index"
[2017-02-16 11:26:38.969] [ERROR] http - ::1 - - "POST /employeeScore HTTP/1.1" 500 3944 "http://localhost:3000/index"
[2017-02-16 11:27:11.825] [ERROR] http - ::1 - - "POST /employeeScore HTTP/1.1" 500 3944 "http://localhost:3000/index"
[2017-02-16 15:14:02.774] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:16:55.791] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:17:32.764] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:23:38.760] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:24:23.328] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:24:30.781] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:31:16.207] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:31:26.143] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:32:58.680] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:33:15.199] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:34:26.182] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:34:54.321] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:35:18.063] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:35:29.005] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:35:32.739] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:35:52.109] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:36:23.213] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:36:37.294] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:36:44.290] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:36:59.410] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:37:43.725] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:37:58.876] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:39:39.724] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:40:11.878] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:40:59.893] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-16 15:41:05.624] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-17 09:36:31.297] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5214 "http://localhost:3000/add"
[2017-02-17 09:37:05.864] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5212 "http://localhost:3000/addWork"
[2017-02-17 10:32:38.535] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:02.634] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:03.124] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:03.477] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:03.675] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:03.904] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:04.127] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-17 10:33:52.705] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2364 ""
[2017-02-17 10:34:35.054] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2365 ""
[2017-02-17 10:35:14.919] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2365 ""
[2017-02-17 10:35:55.575] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2389 ""
[2017-02-17 10:36:12.489] [ERROR] http - ::1 - - "GET /index0 HTTP/1.1" 404 2376 ""
[2017-02-17 14:46:31.567] [ERROR] http - ::1 - - "POST /signup HTTP/1.1" 422 5258 "http://localhost:3000/signup"
[2017-02-18 11:20:01.110] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 3526 "http://localhost:3000/add"
[2017-02-18 13:54:06.367] [ERROR] http - ::1 - - "GET /index2 HTTP/1.1" 404 2375 ""
[2017-02-20 09:51:47.585] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:164:12)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at E:\nodejs\SVN-evaluation-code\evaluation-sys\app.js:65:9
[2017-02-20 09:51:47.601] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 200 5078 "http://localhost:3000/index"
[2017-02-20 10:21:37.612] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:21:37.620] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-20 10:21:37.999] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:21:38.009] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-20 10:21:38.373] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:21:38.382] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-20 10:21:38.739] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:21:38.746] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-20 10:22:26.741] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:22:26.749] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 10:26:29.203] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:26:29.231] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 10:40:39.970] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:40:39.985] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 10:40:42.663] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 10:40:42.677] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 16:44:51.220] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Object.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:20:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
[2017-02-20 16:45:28.382] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 500 2516 "http://localhost:3000/signin"
[2017-02-20 16:45:57.962] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 500 2516 "http://localhost:3000/signin"
[2017-02-20 16:47:44.803] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:164:12)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at E:\nodejs\SVN-evaluation-code\evaluation-sys\app.js:65:9
[2017-02-20 16:47:44.899] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 200 5078 "http://localhost:3000/index"
[2017-02-20 16:47:48.437] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 16:47:48.467] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-20 16:53:09.432] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 16:53:09.512] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-20 17:06:38.077] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 17:06:38.095] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 17:06:39.672] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 17:06:39.682] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 17:06:49.225] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 17:06:49.236] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 17:06:51.123] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-20 17:06:51.133] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - ""
[2017-02-20 19:00:29.564] [ERROR] http - ::1 - - "GET /index1 HTTP/1.1" 404 2375 ""
[2017-02-21 09:48:46.238] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-21 09:48:46.243] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-21 10:22:25.960] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-21 10:22:25.996] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - ""
[2017-02-21 10:23:01.837] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-21 10:23:01.938] [ERROR] http - ::1 - - "GET /add HTTP/1.1" 304 - "http://localhost:3000/index"
[2017-02-21 10:25:54.237] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-21 10:25:54.310] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - ""
[2017-02-21 10:26:18.583] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.contentType (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:552:15)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:139:14)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
[2017-02-21 10:26:18.680] [ERROR] http - undefined - - "GET /add HTTP/1.1" 304 - ""
[2017-02-21 10:29:39.984] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 2518 "http://localhost:3000/add"
[2017-02-21 10:30:36.602] [ERROR] http - ::1 - - "GET /addWork HTTP/1.1" 404 2375 ""
[2017-02-21 10:35:22.235] [ERROR] http - ::1 - - "GET /addWork HTTP/1.1" 404 2375 ""
[2017-02-21 10:39:26.163] [ERROR] http - ::1 - - "GET /addWork HTTP/1.1" 404 2375 ""
[2017-02-21 11:31:50.044] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 2545 "http://localhost:3000/add"
[2017-02-21 11:32:14.338] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 3130 "http://localhost:3000/add"
[2017-02-21 11:32:32.263] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 3182 "http://localhost:3000/add"
[2017-02-21 11:32:41.021] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5225 "http://localhost:3000/add"
[2017-02-21 11:35:29.458] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5241 "http://localhost:3000/add"
[2017-02-21 11:35:31.458] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5241 "http://localhost:3000/addWork"
[2017-02-21 11:35:32.280] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5241 "http://localhost:3000/addWork"
[2017-02-21 11:35:37.668] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 5241 "http://localhost:3000/add"
[2017-02-21 11:36:37.658] [ERROR] http - ::1 - - "GET /addWork HTTP/1.1" 404 2375 ""
[2017-02-21 14:31:22.457] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 3583 "http://localhost:3000/add"
[2017-02-21 15:15:05.489] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 404 2375 "http://localhost:3000/index"
[2017-02-21 15:15:12.825] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2482 "http://localhost:3000/index"
[2017-02-21 15:15:40.491] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2481 "http://localhost:3000/index"
[2017-02-21 15:15:59.128] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2481 ""
[2017-02-21 15:16:54.317] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 3228 ""
[2017-02-21 15:17:41.007] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 3447 ""
[2017-02-21 15:30:23.324] [ERROR] http - ::1 - - "GET /getCompletedWork HTTP/1.1" 404 2375 "http://localhost:3000/getIndexDataByWorkState"
[2017-02-21 15:30:38.475] [ERROR] http - ::1 - - "GET /getCompletedWork HTTP/1.1" 500 3722 ""
[2017-02-21 15:46:37.439] [ERROR] http - ::1 - - "GET /getCompletedWork?wid=25 HTTP/1.1" 500 3560 ""
[2017-02-21 15:49:32.596] [ERROR] http - ::ffff:127.0.0.1 - - "GET /getCompletedWork?wid=25 HTTP/1.1" 500 2820 ""
[2017-02-21 16:01:52.169] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-21 16:06:28.667] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3563 "http://localhost:3000/getCompletedWork?wid=25"
[2017-02-21 16:10:59.564] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-21 16:11:15.809] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 2491 "http://localhost:3000/getCompletedWork?wid=25"
[2017-02-21 16:12:29.482] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3470 "http://localhost:3000/getCompletedWork?wid=25"
[2017-02-21 16:26:06.643] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 2504 "http://localhost:3000/add"
[2017-02-21 16:32:32.959] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3470 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:37:37.588] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3470 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:37:43.816] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 2697 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:38:38.625] [ERROR] http - ::ffff:127.0.0.1 - - "POST /addWorkComplete HTTP/1.1" 500 2683 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:43:37.872] [ERROR] http - ::1 - - "GET /finish_work HTTP/1.1" 404 2375 ""
[2017-02-21 16:43:56.082] [ERROR] http - ::1 - - "GET /getCompletedWork?wid=280 HTTP/1.1" 500 3650 ""
[2017-02-21 16:44:35.388] [ERROR] http - ::1 - - "POST /editWorkRecord HTTP/1.1" 422 5481 "http://localhost:3000/edit?workId=25&uname=sipeng&createTime=1487658747000"
[2017-02-21 16:46:16.941] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3489 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:46:23.178] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3489 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:46:23.778] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3489 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:46:24.021] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3489 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:46:24.292] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3489 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:50:40.878] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-21 16:50:46.247] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3796 "http://localhost:3000/getCompletedWork?wid=28"
[2017-02-21 16:51:01.560] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-21 17:53:57.904] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3675 "http://localhost:3000/getCompletedWork?wid=32"
[2017-02-21 17:59:46.427] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-21 18:00:46.134] [ERROR] http - ::ffff:127.0.0.1 - - "POST /addWorkComplete HTTP/1.1" 500 3603 "http://localhost:3000/getCompletedWork?wid=32"
[2017-02-21 18:00:57.459] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3603 "http://localhost:3000/getCompletedWork?wid=32"
[2017-02-21 18:01:34.296] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3675 "http://localhost:3000/getCompletedWork?wid=32"
[2017-02-22 09:12:20.029] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3794 "http://localhost:3000/getCompletedWork?wid=35"
[2017-02-22 09:12:24.898] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-22 09:46:00.867] [ERROR] http - ::1 - - "GET /edit?workId=35&uname=sipeng&createTime=1487725905000 HTTP/1.1" 500 3905 ""
[2017-02-22 09:46:55.651] [ERROR] http - ::1 - - "GET /edit?workId=35&uname=sipeng&createTime=1487725905000 HTTP/1.1" 500 3850 "http://localhost:3000/index"
[2017-02-22 09:55:53.628] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 3468 "http://localhost:3000/add"
[2017-02-22 09:58:08.018] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 500 3530 "http://localhost:3000/add"
[2017-02-22 09:59:15.652] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 09:59:27.405] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 10:00:01.757] [ERROR] http - ::1 - - "GET /addWork HTTP/1.1" 404 2375 ""
[2017-02-22 10:00:32.429] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 10:00:50.542] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 10:01:16.276] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 10:01:22.791] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 10:01:23.388] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 4568 "http://localhost:3000/add"
[2017-02-22 10:06:10.448] [ERROR] http - ::1 - - "GET /getCompletedWork?wid= HTTP/1.1" 500 3313 ""
[2017-02-22 10:06:10.451] [ERROR] main - Something went wrong: { TypeError: E:\nodejs\SVN-evaluation-code\evaluation-sys\views\finish_work.ejs:18
    16|     <form action="/addWorkComplete" method="post" class="form-container" id="addWorkForm">
    17|         <div class="form-inline">
 >> 18|             工作时间： <%- workPlan.start_worktime %> <br />
    19|             工作计划：
    20|                 <span style="color:#333;"><%- workPlan.work_content %></span>
    21|         </div>

Cannot read property 'start_worktime' of undefined
    at eval (eval at compile (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:481:12), <anonymous>:21:25)
    at returnedFn (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:512:17)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:364:31)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at Query._callback (E:\nodejs\SVN-evaluation-code\evaluation-sys\controllers\work.js:185:12)
    at Query.Sequence.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Sequence.js:86:24)
    at Query._handleFinalResultPacket (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Query.js:137:8)
  path: 'E:\\nodejs\\SVN-evaluation-code\\evaluation-sys\\views\\finish_work.ejs' }
[2017-02-22 10:06:51.561] [ERROR] http - ::1 - - "GET /getCompletedWork?wid=3 HTTP/1.1" 500 3650 ""
[2017-02-22 11:32:11.766] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 3116 "http://localhost:3000/add"
[2017-02-22 11:38:46.823] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3724 "http://localhost:3000/getCompletedWork?wid=39"
[2017-02-22 11:41:23.275] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 3724 "http://localhost:3000/getCompletedWork?wid=39"
[2017-02-22 11:43:08.171] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-22 11:44:13.602] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3953 "http://localhost:3000/getCompletedWork?wid=39"
[2017-02-22 11:44:13.811] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-22 11:44:19.056] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 3953 "http://localhost:3000/getCompletedWork?wid=39"
[2017-02-22 14:17:31.084] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 500 2504 "http://localhost:3000/getCompletedWork?wid=40&uname=sipeng"
[2017-02-22 14:21:33.854] [ERROR] http - ::1 - - "GET /getCompletedWork?wid=40&uname=sipeng HTTP/1.1" 500 3423 ""
[2017-02-22 15:10:46.287] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 4041 "http://localhost:3000/getCompletedWork?wid=41&uname=sipeng"
[2017-02-22 15:17:14.292] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 4443 "http://localhost:3000/getCompletedWork?wid=41&uname=sipeng"
[2017-02-22 15:17:41.733] [ERROR] http - ::1 - - "GET /addWorkComplete HTTP/1.1" 404 2375 ""
[2017-02-22 16:08:15.828] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2520 "http://localhost:3000/index"
[2017-02-22 16:08:23.214] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2520 ""
[2017-02-22 16:08:25.889] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2520 "http://localhost:3000/addWork"
[2017-02-22 16:08:26.791] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2520 ""
[2017-02-22 16:08:29.278] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2520 ""
[2017-02-22 18:11:39.946] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 4335 "http://localhost:3000/getCompletedWork?wid=48&uname=sipeng"
[2017-02-22 18:13:15.251] [ERROR] http - ::1 - - "POST /addWorkComplete HTTP/1.1" 422 4332 "http://localhost:3000/getCompletedWork?wid=39&uname=jony"
[2017-02-22 18:13:36.070] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-22 18:14:20.479] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-22 18:15:23.858] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-02-23 17:21:52.104] [ERROR] http - ::1 - - "POST /updatePasswd HTTP/1.1" 422 2986 "http://localhost:3000/modifyPasswd"
[2017-02-24 14:26:53.935] [ERROR] http - ::1 - - "POST /addWork HTTP/1.1" 422 3104 "http://localhost:3000/add"
[2017-02-27 11:45:30.485] [ERROR] http - ::1 - - "GET /yesterdayWorkTime HTTP/1.1" 404 2375 "http://localhost:3000/summarize"
[2017-02-27 14:34:23.597] [ERROR] http - ::1 - - "POST /editWorkingRecord HTTP/1.1" 500 2500 "http://localhost:3000/getIndexDataByWorkState"
[2017-02-27 14:35:43.965] [ERROR] http - ::1 - - "POST /editWorkingRecord HTTP/1.1" 500 2504 "http://localhost:3000/getIndexDataByWorkState"
[2017-02-27 15:15:12.155] [ERROR] http - ::1 - - "GET /yesterdayWorkTime HTTP/1.1" 404 2375 ""
[2017-02-27 15:44:01.751] [ERROR] http - ::1 - - "POST /yesterdayWorkTime HTTP/1.1" 404 2375 "http://localhost:3000/summarize"
[2017-02-27 15:47:34.171] [ERROR] http - ::1 - - "GET /summarize HTTP/1.1" 500 3160 ""
[2017-02-27 15:50:45.820] [ERROR] summarize_controlllers - undefined
[2017-02-28 15:40:13.647] [ERROR] http - ::1 - - "POST /queryWorkRerordByUserId HTTP/1.1" 500 2500 "http://localhost:3000/index"
[2017-02-28 19:09:20.868] [ERROR] database - error connecting: Error: connect ETIMEDOUT
    at Connection._handleConnectTimeout (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:419:13)
    at Socket.g (events.js:291:16)
    at emitNone (events.js:86:13)
    at Socket.emit (events.js:185:7)
    at Socket._onTimeout (net.js:339:8)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Object.<anonymous> (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:20:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
[2017-02-28 19:09:24.638] [ERROR] http - ::ffff:127.0.0.1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-02-28 19:09:27.622] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 "http://localhost:3000/index"
[2017-02-28 20:21:12.914] [ERROR] http - ::1 - - "GET /sinup HTTP/1.1" 404 2375 ""
[2017-02-28 20:53:40.710] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 403 4184 "http://localhost:3000/signin"
[2017-02-28 20:56:22.796] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 403 4184 "http://localhost:3000/signin"
[2017-02-28 20:58:17.793] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 403 4184 "http://localhost:3000/signin"
[2017-02-28 20:58:20.804] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 403 4184 "http://localhost:3000/signin"
[2017-02-28 20:58:24.517] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 403 4184 "http://localhost:3000/signin"
[2017-02-28 20:58:29.859] [ERROR] http - ::1 - - "POST /signin HTTP/1.1" 422 4169 "http://localhost:3000/signin"
[2017-02-28 21:05:29.481] [ERROR] http - ::1 - - "POST /signup HTTP/1.1" 422 5258 "http://localhost:3000/signup"
[2017-02-28 21:05:36.779] [ERROR] http - ::1 - - "POST /signup HTTP/1.1" 422 5246 "http://localhost:3000/signup"
[2017-03-02 09:16:20.344] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Object.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:20:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
[2017-03-02 09:27:02.359] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Object.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:20:12)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.require (module.js:497:17)
[2017-03-02 11:01:43.385] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:01:44.429] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:35:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:01:47.470] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:01:50.497] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:01:53.531] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:01:56.579] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:02:25.026] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:02:26.043] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:35:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:02:32.320] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Object.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:42:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
[2017-03-02 11:02:35.349] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:02:38.388] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:02:56.859] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:03:01.186] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:03:05.424] [ERROR] http - ::1 - - "GET / HTTP/1.1" 500 2496 ""
[2017-03-02 11:03:26.217] [ERROR] http - ::1 - - "GET / HTTP/1.1" 500 2496 ""
[2017-03-02 11:06:24.049] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:06:25.080] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:35:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:06:28.108] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:06:31.139] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:06:34.161] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:06:37.218] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:07:09.565] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:07:10.116] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:07:10.594] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:35:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:07:11.564] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:07:13.617] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:07:16.652] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:07:27.484] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:07:53.471] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:07:54.499] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:36:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:07:57.533] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:08:12.570] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:08:14.300] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:09:54.029] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:09:55.052] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:36:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:09:58.093] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:10:11.832] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:10:12.851] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:36:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:10:15.895] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:10:18.920] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:10:26.610] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:10:29.256] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:10:30.039] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:24:29.829] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2345 "http://localhost:3000/index"
[2017-03-02 11:24:31.408] [ERROR] http - ::1 - - "GET /getIndexDataByWorkState HTTP/1.1" 500 2345 "http://localhost:3000/index"
[2017-03-02 11:24:36.329] [ERROR] http - ::1 - - "GET / HTTP/1.1" 500 2345 ""
[2017-03-02 11:24:40.886] [ERROR] http - ::1 - - "GET / HTTP/1.1" 500 2345 ""
[2017-03-02 11:24:47.479] [ERROR] http - ::1 - - "GET / HTTP/1.1" 500 2345 ""
[2017-03-02 11:25:27.635] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:25:28.687] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:35:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:25:31.733] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-02 11:25:52.879] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-02 11:25:53.910] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:22:16)
    at Connection.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:35:13)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection._handleProtocolError (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:96:13)
    at Protocol.emit (events.js:188:7)
[2017-03-02 11:26:17.410] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2496 ""
[2017-03-02 11:26:25.085] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2319 ""
[2017-03-02 11:26:25.975] [ERROR] http - ::1 - - "GET /index HTTP/1.1" 500 2319 ""
[2017-03-02 16:14:26.255] [ERROR] invoices_controlllers - 分页查询失败!
[2017-03-02 16:14:26.283] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.send (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:164:12)
    at done (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at E:\nodejs\SVN-evaluation-code\evaluation-sys\app.js:86:9
[2017-03-02 16:14:26.298] [ERROR] http - ::1 - - "POST /invoicePagination HTTP/1.1" 200 43 "http://localhost:3000/invoice"
[2017-03-02 18:00:33.447] [ERROR] http - ::1 - - "POST /addInvoiceInfo HTTP/1.1" 500 3549 "http://localhost:3000/addInvoice"
[2017-03-03 09:05:25.554] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at Object.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:43:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
[2017-03-03 09:05:28.579] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-03 09:05:31.605] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-03 09:05:34.644] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-03 09:05:37.668] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-03 09:06:23.306] [ERROR] http - ::1 - - "GET /invoice HTTP/1.1" 500 2496 ""
[2017-03-04 09:28:16.179] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at Object.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:43:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
[2017-03-04 09:28:19.203] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-04 09:28:22.219] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-04 09:28:25.247] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-04 09:28:28.282] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (E:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:16)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-04 09:39:32.427] [ERROR] http - ::ffff:192.168.1.201 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:39:41.901] [ERROR] http - ::ffff:192.168.1.201 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:40:00.141] [ERROR] http - ::ffff:192.168.1.201 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:40:15.159] [ERROR] http - ::ffff:192.168.1.201 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:40:29.063] [ERROR] http - ::ffff:192.168.1.201 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:40:55.536] [ERROR] http - ::ffff:192.168.1.101 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:41:11.972] [ERROR] http - ::ffff:192.168.1.101 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 09:41:40.528] [ERROR] http - ::ffff:192.168.1.101 - - "POST /signin HTTP/1.1" 403 4184 "http://192.168.1.131:3000/signin"
[2017-03-04 10:09:01.067] [ERROR] http - ::ffff:192.168.1.102 - - "GET /libs/image/fontawesome-webfont.eot HTTP/1.1" 404 2375 "http://192.168.1.131:3000/signin"
[2017-03-04 10:09:07.343] [ERROR] http - ::ffff:192.168.1.102 - - "GET /libs/image/fontawesome-webfont.eot HTTP/1.1" 404 2375 "http://192.168.1.131:3000/index"
[2017-03-04 10:09:16.192] [ERROR] http - ::ffff:192.168.1.102 - - "GET /libs/image/fontawesome-webfont.eot HTTP/1.1" 404 2375 "http://192.168.1.131:3000/index"
[2017-03-04 10:09:30.649] [ERROR] http - ::ffff:192.168.1.102 - - "GET /libs/image/fontawesome-webfont.eot HTTP/1.1" 404 2375 "http://192.168.1.131:3000/index"
[2017-03-05 07:09:18.765] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-05 20:26:32.704] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-06 04:26:33.196] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:188:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-03-07 17:44:24.598] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at Object.<anonymous> (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:43:1)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
[2017-03-07 17:44:27.603] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-07 17:44:30.604] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-07 17:44:33.607] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-07 17:44:36.609] [ERROR] database - error connecting: Error: connect ECONNREFUSED 127.0.0.1:3306
    at Object.exports._errnoException (util.js:1026:11)
    at exports._exceptionWithHostPort (util.js:1049:20)
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1085:14)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at Timeout.connectDb [as _onTimeout] (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
[2017-03-08 11:12:22.068] [ERROR] http - ::1 - - "GET /summarize HTTP/1.1" 500 3171 ""
[2017-03-09 09:25:49.040] [ERROR] http - ::1 - - "GET /invoicePagination HTTP/1.1" 404 2375 ""
[2017-03-10 10:28:26.524] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:30:37.010] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:39:50.817] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:40:25.672] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:41:01.452] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:42:14.953] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:42:40.340] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:43:08.324] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:43:13.942] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:43:24.768] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:43:30.364] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:43:54.968] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:44:25.196] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:44:27.865] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:45:00.098] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:45:05.656] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:46:20.840] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:47:35.906] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:54:18.366] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:54:51.401] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:55:22.876] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 10:55:40.290] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-03-10 14:52:04.971] [ERROR] http - ::1 - - "GET /summarizea HTTP/1.1" 404 2375 ""
[2017-04-26 15:27:58.796] [ERROR] http - ::1 - - "GET /showFurlough HTTP/1.1" 500 3783 "http://localhost:3000/invoice"
[2017-04-26 15:28:22.476] [ERROR] http - ::1 - - "GET /showFurlough HTTP/1.1" 500 3783 "http://localhost:3000/invoice"
[2017-04-26 15:28:56.012] [ERROR] http - ::1 - - "GET /showFurlough HTTP/1.1" 500 3783 "http://localhost:3000/invoice"
[2017-04-26 16:35:38.959] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 5988 "http://localhost:3000/addFurlough"
[2017-04-26 16:47:59.872] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 5990 "http://localhost:3000/addFurlough"
[2017-04-26 16:48:46.667] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3538 "http://localhost:3000/addFurlough"
[2017-04-26 16:49:22.512] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3538 "http://localhost:3000/addFurlough"
[2017-04-26 16:50:01.946] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3494 "http://localhost:3000/addFurlough"
[2017-04-26 17:21:21.977] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3602 "http://localhost:3000/addFurlough"
[2017-04-26 17:22:35.947] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3632 "http://localhost:3000/addFurlough"
[2017-04-26 17:25:50.054] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3522 "http://localhost:3000/addFurlough"
[2017-04-26 17:26:53.950] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3632 "http://localhost:3000/addFurlough"
[2017-04-26 17:27:57.804] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3632 "http://localhost:3000/addFurlough"
[2017-04-26 18:34:28.136] [ERROR] http - ::1 - - "POST /getFurloughAll HTTP/1.1" 500 3778 "http://localhost:3000/furlough"
[2017-04-26 18:35:36.376] [ERROR] http - ::1 - - "POST /getFurloughAll HTTP/1.1" 500 3766 "http://localhost:3000/furlough"
[2017-04-26 19:20:47.140] [ERROR] http - ::1 - - "POST /getFurloughAll HTTP/1.1" 500 3772 "http://localhost:3000/furlough"
[2017-04-26 19:34:19.794] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3666 "http://localhost:3000/addFurlough"
[2017-04-26 19:34:54.982] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3666 "http://localhost:3000/addFurlough"
[2017-04-26 19:35:15.921] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3666 "http://localhost:3000/addFurlough"
[2017-04-26 19:35:17.773] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3666 "http://localhost:3000/addFurlough"
[2017-04-26 19:35:31.848] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 5936 "http://localhost:3000/addFurlough"
[2017-04-26 19:35:39.822] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3666 "http://localhost:3000/addFurlough"
[2017-04-26 19:39:31.982] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3632 "http://localhost:3000/addFurlough"
[2017-04-26 19:40:58.829] [ERROR] http - ::ffff:127.0.0.1 - - "POST /addFurlough HTTP/1.1" 422 5936 "http://localhost:3000/addFurlough"
[2017-04-27 09:29:16.961] [ERROR] http - ::1 - - "GET /addFurlough HTTP/1.1" 500 2381 "http://localhost:3000/furlough"
[2017-04-27 09:29:31.617] [ERROR] http - ::1 - - "GET /addFurlough HTTP/1.1" 500 2381 "http://localhost:3000/furlough"
[2017-04-27 09:29:39.800] [ERROR] http - ::1 - - "GET /addFurlough HTTP/1.1" 500 2381 "http://localhost:3000/furlough"
[2017-04-27 09:29:56.990] [ERROR] http - ::1 - - "GET /addFurlough HTTP/1.1" 500 2836 "http://localhost:3000/furlough"
[2017-04-27 09:30:28.880] [ERROR] http - ::1 - - "GET /addFurlough HTTP/1.1" 500 2381 "http://localhost:3000/furlough"
[2017-04-27 09:30:32.521] [ERROR] http - ::1 - - "GET /addFurlough HTTP/1.1" 500 2381 "http://localhost:3000/furlough"
[2017-04-27 09:41:38.214] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 500 3443 "http://localhost:3000/addFurlough"
[2017-04-27 09:42:08.585] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6728 "http://localhost:3000/addFurlough"
[2017-04-27 09:44:00.197] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6809 "http://localhost:3000/addFurlough"
[2017-04-27 09:44:21.258] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6809 "http://localhost:3000/addFurlough"
[2017-04-27 09:48:27.482] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6809 "http://localhost:3000/addFurlough"
[2017-04-27 09:49:42.943] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6824 "http://localhost:3000/addFurlough"
[2017-04-27 09:49:42.955] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.send (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:164:12)
    at done (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at e:\nodejs\SVN-evaluation-code\evaluation-sys\app.js:95:9
[2017-04-27 09:52:59.836] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6812 "http://localhost:3000/addFurlough"
[2017-04-27 09:55:20.776] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6812 "http://localhost:3000/addFurlough"
[2017-04-27 09:55:24.641] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6812 "http://localhost:3000/addFurlough"
[2017-04-27 09:55:25.664] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6812 "http://localhost:3000/addFurlough"
[2017-04-27 09:55:26.866] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6812 "http://localhost:3000/addFurlough"
[2017-04-27 10:05:53.276] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6810 "http://localhost:3000/addFurlough"
[2017-04-27 10:17:30.883] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:17:37.122] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:18:12.855] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:18:16.366] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:18:38.004] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:19:12.420] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:19:24.082] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:21:03.618] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:23:31.760] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:23:47.836] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:24:29.783] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:24:33.160] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:24:50.588] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:24:58.867] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:25:24.249] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:20.770] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:22.143] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:38.394] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:39.193] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:44.437] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:45.278] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:26:46.867] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:27:19.252] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:27:36.027] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6928 "http://localhost:3000/addFurlough"
[2017-04-27 10:27:36.131] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:27:48.157] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:27:50.531] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:27:54.236] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6928 "http://localhost:3000/addFurlough"
[2017-04-27 10:27:54.336] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:27:57.715] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:28:00.876] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:28:06.863] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:28:09.035] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:28:15.271] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6930 "http://localhost:3000/addFurlough"
[2017-04-27 10:28:15.801] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:08.361] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6949 "http://localhost:3000/addFurlough"
[2017-04-27 10:29:08.545] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:13.932] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6949 "http://localhost:3000/addFurlough"
[2017-04-27 10:29:14.473] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:18.501] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6949 "http://localhost:3000/addFurlough"
[2017-04-27 10:29:19.033] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:31.933] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:39.280] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:40.507] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:29:47.799] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:30:07.625] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:30:24.126] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6947 "http://localhost:3000/addFurlough"
[2017-04-27 10:30:24.257] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:30:29.071] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 6947 "http://localhost:3000/addFurlough"
[2017-04-27 10:30:29.175] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:41:14.263] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:42:35.722] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:42:57.820] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:45:39.298] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:48:40.971] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:48:42.096] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 10:49:48.011] [ERROR] http - ::1 - - "GET /libs/js/jquery-1.10.1.min.map HTTP/1.1" 404 2375 ""
[2017-04-27 11:05:34.132] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 7290 "http://localhost:3000/addFurlough"
[2017-04-27 11:06:09.027] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 7290 "http://localhost:3000/addFurlough"
[2017-04-27 11:06:10.094] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 7290 "http://localhost:3000/addFurlough"
[2017-04-27 11:25:41.626] [ERROR] http - ::1 - - "POST /furloughPagination HTTP/1.1" 500 2488 "http://localhost:3000/furlough"
[2017-04-27 11:25:45.525] [ERROR] http - ::1 - - "POST /furloughPagination HTTP/1.1" 500 2488 "http://localhost:3000/furlough"
[2017-04-27 11:28:26.941] [ERROR] http - ::1 - - "POST /furloughPagination HTTP/1.1" 500 2488 "http://localhost:3000/furlough"
[2017-04-27 11:29:45.413] [ERROR] http - ::1 - - "POST /furloughPagination HTTP/1.1" 500 2497 "http://localhost:3000/furlough"
[2017-04-27 11:30:15.490] [ERROR] main - Something went wrong: { Error: ER_BAD_FIELD_ERROR: Unknown column 'undefined' in 'where clause'
    at Query.Sequence._packetToError (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Sequence.js:52:14)
    at Query.ErrorPacket (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Query.js:77:18)
    at Protocol._parsePacket (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:39:16)
    at Socket.<anonymous> (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:103:28)
    at emitOne (events.js:96:13)
    at Socket.emit (events.js:188:7)
    at readableAddChunk (_stream_readable.js:176:18)
    at Socket.Readable.push (_stream_readable.js:134:10)
    --------------------
    at Protocol._enqueue (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Connection.query (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:208:25)
    at Object.exports.getUserByUserId (e:\nodejs\SVN-evaluation-code\evaluation-sys\models\model_furlough.js:20:20)
    at Query._callback (e:\nodejs\SVN-evaluation-code\evaluation-sys\controllers\furlough.js:221:27)
    at Query.Sequence.end (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Sequence.js:86:24)
    at Query._handleFinalResultPacket (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Query.js:137:8)
    at Query.EofPacket (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\sequences\Query.js:121:8)
    at Protocol._parsePacket (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:280:23)
    at Parser.write (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Parser.js:75:12)
    at Protocol.write (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:39:16)
  code: 'ER_BAD_FIELD_ERROR',
  errno: 1054,
  sqlState: '42S22',
  index: 0 }
[2017-04-27 11:30:15.493] [ERROR] http - ::1 - - "POST /furloughPagination HTTP/1.1" 500 3615 "http://localhost:3000/furlough"
[2017-04-27 11:49:12.175] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 7305 "http://localhost:3000/addFurlough"
[2017-04-27 11:49:12.183] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.send (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:164:12)
    at done (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at e:\nodejs\SVN-evaluation-code\evaluation-sys\app.js:95:9
[2017-04-27 11:49:19.359] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 7305 "http://localhost:3000/addFurlough"
[2017-04-27 11:49:19.364] [ERROR] main - Something went wrong: Error: Can't set headers after they are sent.
    at ServerResponse.OutgoingMessage.setHeader (_http_outgoing.js:356:11)
    at ServerResponse.header (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:719:10)
    at ServerResponse.send (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:164:12)
    at done (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:956:10)
    at View.exports.renderFile [as engine] (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\ejs\lib\ejs.js:369:10)
    at View.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\view.js:126:8)
    at tryRender (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:639:10)
    at EventEmitter.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\application.js:591:3)
    at ServerResponse.render (e:\nodejs\SVN-evaluation-code\evaluation-sys\node_modules\express\lib\response.js:960:7)
    at e:\nodejs\SVN-evaluation-code\evaluation-sys\app.js:95:9
[2017-04-27 14:05:03.773] [ERROR] http - ::1 - - "POST /addFurlough HTTP/1.1" 422 7290 "http://localhost:3000/addFurlough"
[2017-05-08 22:21:13.109] [ERROR] database - error connecting: Error: getaddrinfo ENOTFOUND bfyyg4hrf-mysql.services.clever-cloud.com bfyyg4hrf-mysql.services.clever-cloud.com:3306
    at errnoException (dns.js:26:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:77:26)
    --------------------
    at Protocol._enqueue (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at Object.<anonymous> (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\models\connect-db.js:43:1)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
[2017-05-08 22:26:16.209] [ERROR] database - db error { [Error: Connection lost: The server closed the connection.] fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
Error: Connection lost: The server closed the connection.
    at Protocol.end (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:72:20)
    at Socket.emit (events.js:166:7)
    at endReadableNT (_stream_readable.js:903:12)
    at doNTCallback2 (node.js:439:9)
    at process._tickCallback (node.js:353:17)
[2017-05-08 22:26:16.324] [ERROR] database - error connecting: Error: getaddrinfo ENOTFOUND bfyyg4hrf-mysql.services.clever-cloud.com bfyyg4hrf-mysql.services.clever-cloud.com:3306
    at errnoException (dns.js:26:10)
    at GetAddrInfoReqWrap.onlookup [as oncomplete] (dns.js:77:26)
    --------------------
    at Protocol._enqueue (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:141:48)
    at Protocol.handshake (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\protocol\Protocol.js:52:41)
    at Connection.connect (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:130:18)
    at connectDb (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\models\connect-db.js:23:20)
    at Connection.<anonymous> (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\models\connect-db.js:36:13)
    at emitOne (events.js:77:13)
    at Connection.emit (events.js:169:7)
    at Connection._handleProtocolError (C:\Users\lenovo\Downloads\SVN-evaluation-code\evaluation-sys\node_modules\mysql\lib\Connection.js:433:8)
    at emitOne (events.js:77:13)
    at Protocol.emit (events.js:169:7)
[2017-05-09 09:26:08.976] [ERROR] http - ::1 - - "GET /mahjong/loading.js HTTP/1.1" 404 2391 "http://localhost:3000/replay"
[2017-05-09 09:26:09.148] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 "http://localhost:3000/replay"
[2017-05-09 09:26:11.881] [ERROR] http - ::1 - - "GET /res/favicon.ico HTTP/1.1" 404 2391 "http://localhost:3000/replay"
[2017-05-09 09:26:43.802] [ERROR] http - ::1 - - "GET /mahjong/loading.js HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:26:43.992] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:28:14.425] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 ""
[2017-05-09 09:29:32.732] [ERROR] http - ::1 - - "GET /mahjong/loading.js HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:29:32.985] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:29:35.072] [ERROR] http - ::1 - - "GET /mahjong/favicon.ico HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:30:05.829] [ERROR] http - ::1 - - "GET /mahjong/loading.js HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:30:06.773] [ERROR] http - ::1 - - "GET /HS_9 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:30:43.984] [ERROR] http - ::1 - - "GET /HS_9 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:30:56.896] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-05-09 09:32:05.860] [ERROR] http - ::1 - - "GET /HS_9 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:35:58.402] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-05-09 09:41:27.289] [ERROR] http - ::ffff:192.168.1.131 - - "GET /replay_id=1664&security_s=ujcrxzr9&app_id=71487323064 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:41:49.387] [ERROR] http - ::ffff:192.168.1.131 - - "GET /getReplayData/replay_id=1664&security_s=ujcrxzr9&app_id=71487323064 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:41:58.459] [ERROR] http - ::ffff:192.168.1.131 - - "GET /getReplayData/replay_id=1664&security_s=ujcrxzr9&app_id=71487323064 HTTP/1.1" 404 2391 ""
[2017-05-09 09:47:23.033] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:48:33.678] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:48:38.990] [ERROR] http - ::1 - - "GET /project.json HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:54:47.794] [ERROR] http - ::1 - - "GET /HS_9 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:55:35.962] [ERROR] http - ::1 - - "GET /HS_9 HTTP/1.1" 404 2391 "http://localhost:3000/replay?replay_id=1664&security_s=ujcrxzr9&app_id=71487323064"
[2017-05-09 09:59:05.653] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
[2017-05-09 10:04:09.859] [ERROR] database - db error { Error: Connection lost: The server closed the connection.
    at Protocol.end (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\protocol\Protocol.js:109:13)
    at Socket.<anonymous> (E:\nodejs\sipeng-server-github\sipeng-blog\node_modules\mysql\lib\Connection.js:109:28)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9) fatal: true, code: 'PROTOCOL_CONNECTION_LOST' }
